aos_cd_jobs/updater_test.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+from unittest import TestCase, main
+from mock import Mock, call, patch
+
+from aos_cd_jobs.updater import create_branch, create_remote_branch, get_branch_by_name, list_jobs, update_branches
+
+class TestUpdateMethods(TestCase):
+    def test_list_jobs(self):
+        walker = (
+            ('dir/job0', None, ['Jenkinsfile']),
+            ('dir/notajob', None, ['']),
+            ('dir/job1', None, ['Jenkinsfile']),
+        )
+        self.assertEqual(tuple(list_jobs('dir', walker)), ('job0', 'job1'))
+
+    def test_get_branch_by_name(self):
+        branches = (Mock(), Mock())
+        branches[0].name = 'branch0'
```

Comment:
There doesn't seem to be any added benefit to testing the positive case twice here, so you do not need both of these